### DISCUSSION TITLE
【合作】CommunityOverCode Asia

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [opensource-f2f/episode#230] [General]
推送链接：
* https://twitter.com/osf2f/status/1683750012713594883

官方给出的合作方案

![1690877808722](https://github.com/opensource-f2f/episode/assets/1450685/cab35071-090e-4211-b71b-5f25a364f2fc)

**Comment by LinuxSuRen** [comment]
第二弹
* https://twitter.com/osf2f/status/1685822833379946496
* https://mp.weixin.qq.com/s/-amDm-8OvrY78LzLx1cy7A

**Comment by LinuxSuRen** [comment]
第三弹
* https://twitter.com/osf2f/status/1686626166747357184

**Comment by LinuxSuRen** [comment]
第四弹
* https://twitter.com/osf2f/status/1687322073293873152

**Comment by wjsvec** [comment]
第五弹

- https://twitter.com/osf2f/status/1688489078751477761

**Comment by LinuxSuRen** [comment]
第六弹
* https://twitter.com/osf2f/status/1690156088375836672

**Comment by LinuxSuRen** [comment]
感谢组委会的两张免费赠票，我们已经分别送给了：
* @FeynmanZhou
* @PROPELLER-ZMA

**Reply by wjsvec** [reply]
我看超神经把这两张票在公众号抽了，如果有余票我们以后也可以这么做

**Reply by LinuxSuRen** [reply]
没了，一共两张

**Comment by wjsvec** [comment]
第七弹

- [开源面对面 × CommunityOverCode Asia，不容错过！！！](https://mp.weixin.qq.com/s/JWB73qwnPOXJ92JpbjHJaA)

- https://twitter.com/osf2f/status/1691068785879601152

**Comment by LinuxSuRen** [comment]
小伙伴们在 Apache Con 的合影

![52e0ff1da30e5009d6d2862aab6cd2b](https://github.com/opensource-f2f/episode/assets/1450685/b2f77e51-63a9-44b4-8e25-aca2493dce05)

**Comment by wjsvec** [comment]
期待以后一起和 apache con 合作呀，真的是很棒的活动呢
